server.js

```diff
@@ -1,38 +1,183 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import mongoose from 'mongoose';
+
+import skotrum from './data/skotrum.json';
+
+const mongoUrl =
+  process.env.MONGO_URL || 'mongodb://localhost/skotrum-mongo-project';
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
+
+const Skotrum = mongoose.model('Skotrum', {
+  id: Number,
+  name: String,
+  adress: String,
+  phone: String,
+  openHours: String,
+  note: String,
+  webpage: String,
+  location: String,
+  hasChangingRooms: Boolean
+});
```

Comment:
Awesome to see you really use Mongoose by practicing with your own datasets and working through the fields and deciding types.  Good job.